src/cmd/componentize.js

```
@@ -2,22 +2,38 @@
+ * @param {string} sourcePath - Path to JS source code
+ * @param {object} opts - ComponentizeJS options
+ * @param {string} opts.wit - Path to a WIT file or directory to use
+ * @param {string} [opts.worldName] - Name of the WIT world to target
+ * @param {string} [opts.aot] - Whether to use AoT (via Weval)
+ * @param {string} [opts.engine] - Use a custom engine build (SpiderMonkey/StarlingMonkey) when componentizing
+ * @param {string[]} [opts.disable] - A list of features (i.e. WASI) to disable when building
+ * @param {string} [opts.preview2Adapter] - Path to a custom preview2 adapter
+ * @param {string} opts.out - Path to which to write the WebAssembly component output
+ */
+export async function componentize(sourcePath, opts) {
   const { componentize: componentizeFn } = await eval('import("@bytecodealliance/componentize-js")');
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)